code/datums/actions/mobs/charge.dm

```diff
@@ -51,6 +51,7 @@
 	RegisterSignal(charger, COMSIG_MOVABLE_BUMP, PROC_REF(on_bump), TRUE)
 	RegisterSignal(charger, COMSIG_MOVABLE_PRE_MOVE, PROC_REF(on_move), TRUE)
 	RegisterSignal(charger, COMSIG_MOVABLE_MOVED, PROC_REF(on_moved), TRUE)
+	RegisterSignal(charger, COMSIG_LIVING_DEATH, PROC_REF(charge_end))
```

Comment:
All the other signals are `override = TRUE`'d, but your's is not - do you know if its' necessary or just bad code

Reply:
no clue actually
I'll look into it tomorrow

Reply:
https://github.com/tgstation/tgstation/pull/69419

i guess it also needs the override? then again i don't think i've gotten any warnings on this signal so 🤷

Reply:
You only need it if you register for that same signal elsewhere from the same instance.